doc/getting_started/getting_started.rst

```diff
@@ -79,35 +79,49 @@ To build an example application follow these steps:
 
       $ source zephyr-env.sh
 
-#. Build the :ref:`hello_world` example project, enter:
+#. Generate project files for the :ref:`hello_world` example, enter:
 
    .. code-block:: console
 
       $ cd $ZEPHYR_BASE/samples/hello_world
-      $ make
+      $ mkdir build
```

Comment:
Oh dear, so this means we'll need to find every commandline example where have "make xxx" and change it to mkdir/cd/cmake/cd/ninja throughout the documentation, right?

Reply:
Yes, or the slightly shorter

cmake -H. -Bbld && cmake --build bld